bigbluebutton-html5/imports/api/audio/client/bridge/kurento.js

```diff
@@ -126,7 +133,9 @@ export default class KurentoAudioBridge extends BaseAudioBridge {
         await audioContext.setSinkId(value);
         this.media.outputDeviceId = value;
       } catch (err) {
-        logger.error({ logCode: 'audio_kurento_changeoutputdevice_error' }, err);
+        logger.error({ logCode: 'audio_kurento_changeoutputdevice_error', err });
+        logFunc('error', 'SFU audio bridge failed to fetch STUN/TURN info, using default',
+          { logCode: 'audio_kurento_changeoutputdevice_error', err });
```

Comment:
Why is `logger.error` and `logFunc` called together? Don't they both go to the same place?